ui/src/chatbot.tsx

```diff
@@ -91,12 +97,14 @@ export interface Chatbot {
   data: Rec
   /** Chat input box placeholder. Use for prompt examples. */
   placeholder?: S
-  /** The events to capture on this chatbot. One of 'stop' | 'scroll_up'. */
+  /** The events to capture on this chatbot. One of 'stop' | 'scroll_up' | 'feedback'. */
   events?: S[]
   /** True to show a button to stop the text generation. Defaults to False. */
   generating?: B
   /** The previous messages to show as the user scrolls up. */
   prev_items?: ChatbotMessage[]
+  /** True to show thumbs up/down buttons to capture the feedback event on chatbot response. Defaults to False. */
+  feedback?: B
```

Comment:
Isn't registering `feedback` event enough?

Reply:
Makes sense, it would be easier in terms of usage.